PubnativeLite/PubnativeLite/MRAID/HyBidMRAIDView.m

```diff
@@ -793,6 +793,9 @@ - (void)addContentInfoViewToView:(UIView *)view {
     contentInfoView.delegate = self;
     [view addSubview:contentInfoViewContainer];
     [contentInfoViewContainer addSubview:contentInfoView];
+    
+    [[HyBidViewabilityWebAdSession sharedInstance] addFriendlyObstruction:contentInfoViewContainer toOMIDAdSession:adSession withReason:@"This view is related to Content Info" isInterstitial:isInterstitial];
```

Comment:
yes, contentInfoView is subview of the container.

Reply:
about your question, we didnt had any issues with VAST. The problem is only with banner/native as gaylord mentionned.

Reply:
I've asked VAST because, in `PNLiteVASTPlayerViewController.m` both `contenInfoViewContainer` and `contentInfoView` is added as a friendly obstruction.